.env-sample

```diff
@@ -24,6 +24,9 @@ FIRE_PROJECT_ID={firebase project id}
 
 # If using mysql, enter your config here
 
+# DATABASE_URL={mysql://db_address}
```

Comment:
```suggestion
DATABASE_URL={mysql://db_address}
```

Reply:
we likely want to keep the variables as the default - the nature of this PR should not change what we've been doing